app/views/helpers/feed/update.phtml

```diff
@@ -125,6 +126,10 @@
 						echo '<option value="' . intval($this->feed->ttl()) . '" selected="selected">' . intval($this->feed->ttl()) . 's</option>';
```

Comment:
Isn't there a little bug there?
When the feed is muted, and we edit it, then no TTL option will be selected (the browser will pick the first one in the list). When we save, it looks like we will override the TTL with a wrong value.

One option could be to show the old value.

Reply:
That how it's done. The TTL is an absolute value of the field in the DB. It is splitted in the ttl and mute fields. So if you have ```-900``` in db, you'll have ```900``` in ttl and ```true``` in mute.